packages/bolt-interactions/src/types/interaction.ts

```diff
@@ -0,0 +1,33 @@
+import { App, SlackAction, SlackActionMiddlewareArgs } from '@slack/bolt';
```

Comment:
So it turns out that by default custom `.d.ts` files aren't built. 🤷‍♂ They're intended to be a "hint to the compiler". SO, to export types, they have to be `.ts` files. I made that change, and converted it to a module to remove the namespace.

Reply:
This is not related to the pr, so, a bit of boy scouting 😅

Reply:
It makes sense that `.t.s` files are excluded by default. If we want to include them shouldn't we just use a compiler flag? 

or I guess have a `main.d.ts` that we point the `types` property to in our `package.json` that grabs the rest.

https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package

Reply:
There isn't a compiler flag to copy hand crafted ones. We could have a post-build step that did it, but that seemed like more work than renaming.